app/api-v1/controllers/Recipe/index.js

```diff
@@ -0,0 +1,50 @@
+const { runProcess } = require('../../../utils/PolkadotApi')
+const { client } = require('../../../db')
+const { mapRecipeData } = require('./helpers')
+const { BadRequestError, NotFoundError } = require('../../../utils/errors')
+
+module.exports = {
+  transaction: {
+    create: async (req) => {
+      const service = '/recipe/{id}/creation'
+      const { id } = req.params
+      if (!id)
+        throw new BadRequestError({
+          message: 'missing [id] param from request',
+          service,
+        })
+
+      const recipe = await client.from('recipes').select('*').where({ id })
+      if (!recipe || !recipe.length)
+        throw new NotFoundError({
+          message: 'recipe not found',
+          service,
+        })
+
+      const token = await runProcess({
+        inputs: [],
+        outputs: [
+          {
+            roles: { Owner: '5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY' },
```

Comment:
Hardcoded?

Reply:
oh... this should not here. basically polkadotApi expects some role so I guess this is a questions whether this will be saved in the database along other metadata or not, if so, then II should pull it from there @mattdean-digicatapult @danielcooperxyz ?

Reply:
Added new property to recipe `role_address`